packages/framework/src/Model/SocialNetwork/SocialNetworkFacade.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Model\SocialNetwork;
+
+use App\Model\Customer\User\CustomerUserFacade;
+use App\Model\Customer\User\RegistrationDataFactory;
+use App\Model\Customer\User\RegistrationFacade;
+use App\Model\Security\LoginAsUserFacade;
```

Comment:
In the framework package (namespace), you must not use classes from the project-base (app namespace). It will not work outside of the monorepo context. You may be forced to move some of the classes from the project-base to the framework package first, we can discuss this :wink: